README.md

```diff
@@ -138,6 +138,14 @@ render json: @post, meta: { total: 10 }, meta_key: "custom_meta"
 
 `meta` will only be included in your response if you are using an Adapter that supports `root`, as JsonAPI and Json adapters, the default adapter (FlattenJson) doesn't have `root`.
 
+### Links
```

Comment:
As such, what do you think about the changes that add interfaces to the serializers to support the adapter, e.g. the `Serializer#links` attribute below and my comment in https://github.com/rails-api/active_model_serializers/pull/1041#discussion_r36708460 ? This decision has kind of big implications on the development and extension of the library (IMHO), so I want to make sure we've thought through how it works internally.

e.g. `render json: @profiles, links: { self: "/profiles/1" }` suggests to me that the `links` option should be passed to the adapter in the [SerializableResource](https://github.com/rails-api/active_model_serializers/blob/03ddeeeec1e4eedafeb68eeccd59e8c3826b859d/lib/active_model/serializable_resource.rb#L5). I'd even consider that meta should be passed to the adapter as well.